jumpscale/sals/nginx/nginx.py

```diff
@@ -114,26 +115,48 @@ def get_config(self):
 
     def generate_certificates(self):
         if self.ssl:
-            j.sals.process.execute(
+            res = j.sals.process.execute(
```

Comment:
how would you know if it was a success? it returns a tuple of (rc, out, err). you need to explicitly check there and raise if needed no? and die is set to False by default already so you wouldn't know.